tests/models/transformers/test_models_transformer_aura_flow.py

```diff
@@ -71,3 +71,7 @@ def prepare_init_args_and_inputs_for_common(self):
         }
         inputs_dict = self.dummy_input
         return init_dict, inputs_dict
+
+    @unittest.skip("AuraFlowTransformer2DModel uses its own dedicated attention processor. This test does not apply")
+    def test_set_attn_processor_for_determinism(self):
```

Comment:
maybe we can add a `default_attn_processor` attribute to the model classes now a lot of these new transformers we added have their own attention processors